spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/TcpConfigOutboundGatewayTests-context.xml

```diff
@@ -37,6 +38,9 @@
 							   serializer="crLfSerializer"
 							   deserializer="crLfSerializer"/>
 
+	<ip:tcp-inbound-gateway id="gatewayCrLf2"
+		connection-factory="crLfServer2"
+		request-channel="toSA" />
```

Comment:
Got it! 	
Nasty bug, BTW...

I haven't pulled it locally yet for merging, but I wonder if that Travis failure is a bell for you.